src/entities/trades/curve/CurveTrade.ts

```diff
@@ -184,8 +184,8 @@ export class CurveTrade extends Trade {
     const nativeCurrency = Currency.getNative(chainId)
 
     //check if tokens are not from official list
-    const areMuffTokens = curveTokenIn != undefined || curveTokenIn != undefined
-    console.log('fucking muffs', areMuffTokens)
+    const areMuffTokens = curveTokenIn === undefined || curveTokenIn === undefined
+    console.log('fucking muffs 1', areMuffTokens)
```

Comment:
Language!

Reply:
I just found out that we need to extrapolate those 2crv and 3crv pools to their underlying tokens...This can push back a bit since we can't fetch those tokens from the api...